dags/openshift_nightlies/models/release.py

```diff
@@ -34,7 +34,7 @@ def _get_git_user(self):
 
     def _generate_cluster_name(self):
         git_user = self._get_git_user()
-        git_branch = environ['GIT_BRANCH']
+        git_branch = str(environ['GIT_BRANCH']).replace("_","-")
```

Comment:
Cool. Do we also need to lower-case any upper-case?

Reply:
🤔

Reply:
Yes, I think so.